### ISSUE TITLE
Error "searchEles.childNodes" with version 0.0.8

### ISSUE THREAD

**Issue by gamer06** [custom-cards/vertical-stack-in-card#32]
Hi, 

I have the following configuration and got this error in home assistant
Everything works fine but I get this error
https://myip:***/local/custom_ui/vertical-stack-in-card.js?v=0.0.8:58:44 TypeError: null is not an object (evaluating 'searchEles.childNodes')

parts of my config:

  - url: /local/custom_ui/vertical-stack-in-card.js?v=0.0.8
    type: js

  - id: 1
    icon: mdi:home-assistant
    name: Casa
    cards:
      - type: vertical-stack
        cards:
          - type: custom:vertical-stack-in-card
            title: Casa
            cards:
              - type: glance
                entities:
                  - alarm_control_panel.house
                  - sensor.disarmed_home_away
                  - sensor.interruptorcamera
                  - switch.interruptor_camera
              - type: entities
                entities:
#                    - type: section
#                    - type: divider
                  - entity: binary_sensor.sensor_porta_principal
                    secondary_info: last-changed
                  - entity: sensor.porta_rua_time
                  - type: section
                  - entity: binary_sensor.corredor
                    secondary_info: last-changed
                  - entity: sensor.interruptorsirene
                    secondary_info: last-changed
                  - entity: switch.interruptor_sirene

**Comment by gamer06** [comment]
Any news about that? thanks

**Comment by ofekashery** [comment]
Is this still happening to you? It works for me

**Comment by gamer06** [comment]
Yes. I’m still with HA 0.82.1 but when I got home i will try updating to 0.83.1

**Comment by gamer06** [comment]
Yes...Still the same:

https://gyazo.com/e96376cc5b38acf939669f01ed1fe77b

**Comment by hoopsta1423** [comment]
Just started happening to me as well on HA 0.83.2. Was fine for an hour or so after I created the card but those errors just randomly started popping up now
